src/Resources/public/Admin.js

```diff
@@ -688,6 +688,35 @@ var Admin = {
                 form.find('button').prop('disabled', true);
             }, 1);
         });
+    },
+    /**
+     * Remeber open tab after refresh page.
+     */
+    setup_view_tabs_changer: function() {
+        jQuery('.changer-tab').on('click', function () {
+            var tab = jQuery(this).attr('aria-controls'),
+                search = location.search.substring(1);
+
+            /* Get query string parameteres from URL */
+            var parameters = decodeURIComponent(search).replace(/"/g, '\\"').replace(/&/g, '","').replace(/=/g,'":"'),
+                jsonURL = '{}';
+
+            /* If the parameters exist and their length is greater than 0, we put them in json */
+            if(parameters.length) {
```

Comment:
Please use jshint or jslint, there should be a space before this opening parenthesis